server/src/main/java/com/groep9/moviemingleserver/data/UserDAO.java

```diff
@@ -53,9 +53,8 @@ public User get(long id) {
      * @return the user if found, null otherwise
      */
     public User get(String email) {
-        Connection conn = DatabasePool.getConnection();
 
-        try {
+        try (Connection conn = DatabasePool.getConnection()) {
```

Comment:
replace with try with resources to close the connection, otherwise we might hit a connection leak